doc/source/data/data.rst

```diff
@@ -117,7 +117,7 @@ Advanced users can refer directly to the Ray Data :ref:`API reference <data-api>
         Learn how to :ref:`load data <loading_data>`, :ref:`save
         data <saving_data>`, :ref:`transform data <transforming_data>`,
         :ref:`access and exchange data <consuming_data>`, or
-        :ref:`work with tensor data <data_tensor_support>`.
+        :ref:`work with tensor data <working_with_tensors>`.
```

Comment:
Thanks!

Reply:
sure! strangely the warning for this was only surfaced after I introduced a typo in the other ref... we might have more of those